src/libs/Formula.ts

```diff
@@ -402,19 +402,37 @@ function formatDate(dateString: string | undefined, format = 'yyyy-MM-dd'): stri
 /**
  * Format an amount value
  */
-function formatAmount(amount: number | undefined, currency: string | undefined): string {
+function formatAmount(amount: number | undefined, currency: string | undefined, format?: string): string {
```

Comment:
@mohammadjafarinejad `format` doesn't seem like a very good name for the parameter. Should we use something like `displayCurrency`?